docs/Styling2ndGen.md

```diff
@@ -13,17 +13,17 @@ Bloom design tokens include colors, typography settings, sizes, borders, and so
 For example, some colors in `tokens/colors.scss`:
```

Comment:
Github won't let me comment on the line above this, but we should probably update this file to have links to the new ui-components repo. Particularly lines 3 and 11